services/node-services/README.md

```diff
@@ -14,10 +14,19 @@ To build:
 $ npm run build
 ```
 
-To build the docker image:
+## Build and push the docker image:
+A node services Docker image is used by the verification tests in Kubernetes.
 
 ```shell
-$ gradle :services:node-services:dockerBuild
+$ docker build --platform linux/arm64,linux/amd64 -t ghcr.io/restatedev/e2e-node-services --push .
+```
+
+## Update the Lambda function
+A node services Lambda is used by the Lambda verification tests.
+
+```shell
+npm run bundle
+aws lambda update-function-code --function-name 'arn:aws:lambda:eu-central-1:663487780041:function:e2e-node-services' --zip-file fileb://dist/index.zip
```

Comment:
Maybe also state when to create a new release. I assume it is required every time a breaking change happens in one of the SDKs.

How will we prevent this from being forgotten (like me doing a change in the SDK and then forgetting about it)?

Reply:
well, if its a breaking change, you know to do it when the tests start failing, which is generally when i do it. if it was some other subtle sdk change that we wanted to test in the verif cluster, we would need to automate this to truly avoid it. i dont think its really that important right now though

Reply:
Fair enough. Merging this PR now.